projects/fluid_simulation/sim.cpp

```diff
@@ -238,7 +238,7 @@ void FluidSimulation::InitBloomTextures()
     ppx::int2 res = GetResolution(GetConfig().pBloomResolution->GetValue());
     mBloomTexture = std::make_unique<Texture>(this, "bloom", res.x, res.y, kRGBA);
     PPX_ASSERT_MSG(mBloomTextures.empty(), "Bloom textures already initialized");
-    for (uint32_t i = 0; i < GetConfig().pBloomIterations->GetValue(); i++) {
+    for (int i = 0; i < GetConfig().pBloomIterations->GetValue(); i++) {
```

Comment:
Curious why the change from `uint32_t` to `int`?

Reply:
Warning about different signedness. The knob returns int.